internal/dialect/dialectquery/postgres.go

```diff
@@ -40,3 +50,17 @@ func (p *Postgres) GetLatestVersion(tableName string) string {
 	q := `SELECT max(version_id) FROM %s`
 	return fmt.Sprintf(q, tableName)
 }
+
+func (p *Postgres) TableExists(tableName string) string {
+	schemaName, tableName := parseTableIdentifier(tableName)
```

Comment:
This should handle cases where the table name contains the schema name; otherwise, assumed `public.`